tutorials/advanced_features/Output Modulation and EOM Mode.ipynb

```diff
@@ -0,0 +1,317 @@
+{
```

Comment:
<p>Has this new way of creating virtual devices been replaced everywhere across the documentation?</p>
<br/>

 _Reply via <a href='https://app.reviewnb.com/pasqal-io/Pulser/pull/421/discussion/'>ReviewNB</a>_ <div id='ReviewNBCommentContext-DoNotDelete' style='display:none' data-state='OPEN' data-cellIndex='14'/>

Reply:
I meant that you are now using the new `VirtualDevice` which is quite recent. I wanted to understand if this has been replaced across the whole documentation even though backward compatibility is maintained.

Reply:
There is no tutorial explaining how to create a `VirtualDevice` (nor a `Device`, for that matter), so I think there is nothing to replace because this is just another way of getting a device for your sequence. 
That being said, the distinction between the two is already explained in the [docs](https://pulser.readthedocs.io/en/latest/apidoc/core.html#structure-of-a-device).